src/l500/CMakeLists.txt

```diff
@@ -21,3 +21,7 @@ target_sources(${LRS_TARGET}
         "${CMAKE_CURRENT_LIST_DIR}/l500-serializable.h"
         "${CMAKE_CURRENT_LIST_DIR}/l500-options.h"
 )
+
+if (ENABLE_DEPTH_INVALIDATION)
+    add_definitions(-DDEPTH_INVALIDATION_ENABLED)
```

Comment:
I suggest you keep the define name the same as the CMake variable to make it more intuitive to search

Reply:
agreed.